Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,110 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+ /* General */
+
+body{
+    font-family: Helvetica, sans-serif;
+}
+
+/* Topbar */
+.topbar{
+    height:70px;
+    width:100%;
+}
+
+.logo{
+    margin-left:20px;
+}
+/* big image section */
+.top-container{
+    display: flex;
+    flex-direction: column; 
+    justify-content: center;
+    align-items: center;
+    background-image: url('../images/flowershop.jpg');
+    background-position: center center ;
+    background-size:cover;
+    background-repeat: no-repeat;
+    height: 720px;
+
+}
+
```

Comment:
Superb use of an overlay to make the banner image darker!
You could use linear gradient with RGBA colour.